specs/accounting_reform/beacon-chain.md

```diff
@@ -0,0 +1,278 @@
+# Incentive accounting  reforms
+
+## Table of contents
+
+**TODO**: Generate table of contents with doctoc.
+
+## Introduction
+
+This is a proposed incentive accounting simplification which replaces `PendingAttestation` with epoch participation flags.
+
+## Constants
+
+### Participation flags
+
+| Name | Value |
+| - | - |
+| `TIMELY_HEAD_FLAG` | `0` |
+| `TIMELY_SOURCE_FLAG` | `1` |
+| `TIMELY_TARGET_FLAG` | `2` |
+
+### Participation rewards
+
+| Name | Value |
+| - | - |
+| `TIMELY_HEAD_NUMERATOR` | `12` |
+| `TIMELY_SOURCE_NUMERATOR` | `12` |
+| `TIMELY_TARGET_NUMERATOR` | `32` |
+| `REWARD_DENOMINATOR` | `64` |
+
+The reward fractions add up to 7/8, leaving the remaining 1/8 for proposer rewards and other future micro-rewards.
+
+### Misc
+
+| Name | Value |
+| - | - |
+| `PARTICIPATION_FLAGS_LENGTH` | `8` |
```

Comment:
Why is this 8 and not 3?

Reply:
Yeah, think it should be 3
Unless @vbuterin is padding out for future flags, but I don't think that's very valuable because we'd have to hard-fork to add them anyway